fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/data/FreePanic.scala

```diff
@@ -0,0 +1,146 @@
+package izumi.functional.bio.data
+
+import izumi.functional.bio.{BIOExit, BIOPanic}
+
+import scala.util.Try
+
+sealed trait FreePanic[S[_, _], +E, +A] {
```

Comment:
Hmm, I feel a bit sad that the `S` is invariant, but if we change it to `FreePanic[+S[_, _], +E, +A]` this will cause every method to have a noisy `[S1[e, a] >: S[e, a]` parameter.
Could be worth it tho, it would allow us to auto-widen to a union of functors when we get unions (in a couple of years...) or if we use a tagless union encoding (https://medium.com/@odomontois/tagless-unions-in-scala-2-12-55ab0100c2ff)

Reply:
@neko-kai I've updated S type to be covariant for both `Free` and `FreePanic`. Don't sure if it's really needing but why not.